modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -397,6 +387,74 @@ private void listenForTableChange() {
 
     /** {@inheritDoc} */
     @Override public Table table(String name) {
-        return tables.get(name);
+        if (!isTableConfigured(name))
+            throw new IgniteException("Table wasn't found [name=" + name + ']');
```

Comment:
Inconsistent with javadoc

> Tables with corresponding name or {@code null} if table isn't created.

Reply:
I change this behavior as same as in Ignite 2.
It returns null if the table does not exist.